spec/lib/active_graphql/model/relation_proxy_spec.rb

```diff
@@ -151,7 +151,35 @@ def self.name
         it 'builds graphql with ASC order direction' do
           expect(order.to_graphql).to eq <<~GRAPHQL
             query {
-              users(order: { by: SOMETHING, direction: ASC }) {
+              users(order: [{ by: SOMETHING, direction: ASC }]) {
+                edges { node { id, firstName } }, pageInfo { hasNextPage }
+              }
+            }
+          GRAPHQL
+        end
+      end
+
+      context 'when multiple fields are given' do
+        subject(:order) { relation_proxy.order(:something, something1: :asc, something2: :desc) }
```

Comment:
please add another test for multiple orders case:
`relation_proxy.order(:something).order(something1: :asc)`